src/Tribe/Aggregator/REST/V1/Endpoints/State.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+class Tribe__Events__Aggregator__REST__V1__Endpoints__State
+	extends Tribe__Events__Aggregator__REST__V1__Endpoints__Base
+	implements Tribe__REST__Endpoints__CREATE_Endpoint_Interface {
+
+	/**
+	 * Handles POST requests on the endpoint.
+	 *
+	 * @param WP_REST_Request $request
+	 * @param bool $return_id Whether the created post ID should be returned or the full response object.
+	 *
+	 * @return WP_Error|WP_REST_Response|int An array containing the data on success or a WP_Error instance on failure.
+	 */
+	public function create( WP_REST_Request $request, $return_id = false ) {
+		/** @var Tribe__Events__Aggregator__Records $records */
+		$records = tribe( 'events-aggregator.records' );
+
+		/** @var Tribe__Events__Aggregator__Record__Abstract $record */
+		$record = $records->get_by_import_id( $request['import_id'], array( 'post_status' => 'any' ) );
+
+		$status = $request['status'];
+
+		$updated = $record->set_status( $status );
+		$record->update_meta( 'done', $request['done'] );
```

Comment:
Should this line be executed if `$updated` is an error state (empty or WP_Error)? Or should it be pushed down to be executed around line 35-ish?